src/iop/clipping.c

```diff
@@ -1957,7 +1950,19 @@ void gui_update(struct dt_iop_module_t *self)
     keystone_type_populate(self, FALSE, p->k_type);
   }
 
-  aspect_presets_changed(g->aspect_presets, self);
+  /* special handling the combobox when current act is already selected
+     callback is not called, let do it our self then..
+   */
+  if(act == -1)
+  {
+    char str[128];
+    snprintf(str, sizeof(str), "%d:%d", abs(p->ratio_d), p->ratio_n);
```

Comment:
please use `snprintf(str, sizeof(str), "%d:%d %2.2f", abs(p->ratio_d), p->ratio_n, (float)p->ratio_d / (float)p->ratio_n);` to be consistent with `aspect_preset_changed()`

Reply:
ok

Reply:
done